ingest.py

```
@@ -0,0 +1,177 @@
+from logger import setup_logger
+
+logger = setup_logger(__name__)
+
+
+def embed_text(documents: list[Document]):
+    embeddings = AzureOpenAIEmbeddings(
+        azure_deployment=env.embeddings_model_name,
+        chunk_size=1,
+    )
+
+    logger.info(f"Generating embeddings.")
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)